packages/flutter_tools/test/general.shard/fuchsia/fuchsia_device_test.dart

```diff
@@ -57,20 +57,19 @@ void main() {
 
```

Comment:
Is there already a test with multiple devices from `FuchsiaDevFinder.list()`?

Reply:
Test added